src/webextension/widgets/scrolling-list.css

```diff
@@ -14,16 +14,6 @@
   outline-style: none;
 }
 
-.styled-item {
```

Comment:
We should keep all this stuff here. Instead, we should change how `styledItems` is set. Right now, it's set in `src/webextension/list/manage/containers/all-items.js`; I think it should go in `src/webextension/list/components/item-list.js` so that it applies to all instances of the item list.

That does mean these CSS rules will be unused for now, but I expect we'll want this the way it is when we add support for multiple origins in the editor.

Reply:
I just want to verify this is the correct implementation of this change:
`src/webextension/list/manage/containers/all-items.js`
Screen Shot 2018-03-08 at 11.46.39 AM

`src/webextension/list/components/item-list.js`
<img width="939" alt="screen shot 2018-03-08 at 11 46 03 am" src="https://user-images.githubusercontent.com/5185090/37169930-9dabd810-22c6-11e8-965d-a56d1bed58fc.png">

Reply:
I'll have to look at the full diff, but the only thing that jumps out here is that `style.allItems` should be `styles.itemList`. Also, don't forget to update /src/webextension/list/**popup**/containers/all-items.js similarly.

Reply:
I'm unsure of what needs changed for the `src/webextension/list/popup/containers/all-items.js` file. Can you let me know what needs to be updated here when you have a moment?

Reply:
It might actually be ok as it is. I thought we had some special styles in there, but it looks like no... vOv